androidScanner/app/src/main/java/com/example/androidScanner/App.java

```diff
@@ -0,0 +1,30 @@
+package com.example.androidScanner;
+
+import android.app.Application;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.os.Build;
+
+public class App extends Application {
+
+    private static final String CHANNEL_ID = "exampleChannel1";
+    private final static int REQUEST_ENABLE_BT = 1;
+    private static final int PERMISSION_REQUEST_COARSE_LOCATION = 1;
+
+    public void createNotificationChannel() {
+
+        NotificationChannel serviceChannel = new NotificationChannel(CHANNEL_ID, "example", NotificationManager.IMPORTANCE_LOW);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
If minsdk is already 26 then ain't this bit recursive because doesn't that mean you cannot use this program if it's older then android O?